app/routes/goal_routes.py

```diff
@@ -1 +1,57 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, abort, make_response
```

Comment:
`abort` and `make_response` are imported but not accessed so they should be removed.